File: config/plugins/claude-master-suite/scripts/python/llms_generator.py
import os
import sys

def main():
    if len(sys.argv) < 2:
        print("Kullanım: python llms_generator.py <hedef_klasor>")
        sys.exit(1)
        
    target_dir = sys.argv[1]
    
    llms_content = """# Domain
Projeniz Hakkında Genel Bilgi

> Summary
Bu dosya yapay zeka ajanları ve LLM'ler için tasarlanmıştır.

## Mimariler ve Alt Linkler
- [Core Architecture](/llms/core.md)
- [Sayfalar](/llms/pages/index.md)
"""

    llms_dir = os.path.join(target_dir, 'llms')
    llms_pages_dir = os.path.join(llms_dir, 'pages')
    
    try:
        os.makedirs(llms_pages_dir, exist_ok=True)
        
        with open(os.path.join(target_dir, 'llms.txt'), 'w', encoding='utf-8') as f:
            f.write(llms_content)
            
        with open(os.path.join(llms_dir, 'core.md'), 'w', encoding='utf-8') as f:
            f.write('# Core System\nLLM Yönergeleri buraya...')
            
        with open(os.path.join(llms_pages_dir, 'index.md'), 'w', encoding='utf-8') as f:
            f.write('# Pages\nSayfa alt kırılımları...')
            
        print("\n=== SÜPER SEO MOTORU (LLMS HUB OLUŞTURUCU V3.0) ===")
        print(f"[+] BAŞARILI: {os.path.abspath(target_dir)} dizinine `llms.txt` hub ve alt dizinleri oluşturuldu.")
    except Exception as e:
        print(f"[X] HATA: {e}")

if __name__ == "__main__":
    main()

File: config/plugins/codex-master-suite/scripts/llms-generator.js
const fs = require('fs');
const path = require('path');

const targetDir = process.argv[2];
if (!targetDir) {
    console.log("Kullanım: node llms-generator.js <hedef_klasor>");
    process.exit(1);
}

const llmsContent = `# Domain
Projeniz Hakkında Genel Bilgi

> Summary
Bu dosya yapay zeka ajanları ve LLM'ler için tasarlanmıştır.

## Mimariler ve Alt Linkler
- [Core Architecture](/llms/core.md)
- [Sayfalar](/llms/pages/index.md)
`;

const llmsDir = path.join(targetDir, 'llms');
const llmsPagesDir = path.join(llmsDir, 'pages');

try {
    if (!fs.existsSync(llmsDir)) fs.mkdirSync(llmsDir, { recursive: true });
    if (!fs.existsSync(llmsPagesDir)) fs.mkdirSync(llmsPagesDir, { recursive: true });
    
    fs.writeFileSync(path.join(targetDir, 'llms.txt'), llmsContent);
    fs.writeFileSync(path.join(llmsDir, 'core.md'), '# Core System\nLLM Yönergeleri buraya...');
    fs.writeFileSync(path.join(llmsPagesDir, 'index.md'), '# Pages\nSayfa alt kırılımları...');
    
    console.log(`\n=== SÜPER SEO MOTORU (LLMS HUB OLUŞTURUCU V3.0) ===`);
    console.log(`[+] BAŞARILI: ${path.resolve(targetDir)} dizinine \`llms.txt\` hub ve alt dizinleri oluşturuldu.`);
} catch (err) {
    console.error(`[X] HATA: ${err.message}`);
}
